apps/src/templates/VersionRow.jsx

```diff
@@ -0,0 +1,48 @@
+var React = require('react');
+
+/**
+ * A single row in the VersionHistory dialog, describing one version of a project.
+ */
+module.exports = React.createClass({
+  propTypes: {
+    lastModified: React.PropTypes.instanceOf(Date),
+    isLatest: React.PropTypes.bool,
+    onChoose: React.PropTypes.func
+  },
+
+  getTimestamp: function () {
```

Comment:
Could make this a pure function and take this.props.lastModified as an input. Right now this function is really `getlastModifiedTimestamp`